### PR TITLE
MR metrics, use separate names for job level and task level metrics 

### PR DESCRIPTION
**Pull request by gsps1**
Since Standalone doesn't have any tasks, aggregating task level metrics for a job as job level metrics wouldn't work.
so having separate metric names for job and task level metrics. 

In UI code, we were using number of tasks and the number of tasks completed to calculate the percentage completion, since the numTasks is always zero in standalone, completion percentage is also always zero.

mapProgress and reduceProgress returned by the info endpoint is correct and using that to show the progress completion instead.
https://issues.cask.co/browse/CDAP-2138

### CODE REVIEW

**Comment by anwar6953** [comment]
Is the `getCompletedPercentage` function still in use, then?

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceMetricsWriter.java [hunk 1]
```diff
@@ -95,7 +95,15 @@ private void reportMapredStats() throws IOException, InterruptedException {
     int memoryPerMapper = jobConf.getConfiguration().getInt(Job.MAP_MEMORY_MB, Job.DEFAULT_MAP_MEMORY_MB);
     int memoryPerReducer = jobConf.getConfiguration().getInt(Job.REDUCE_MEMORY_MB, Job.DEFAULT_REDUCE_MEMORY_MB);
 
+    Counters counters = jobConf.getCounters();
```
Comment (anwar6953) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceMetricsWriter.java]: If this is not local (rpc call?), then just use the same instance that's used in `reportSystemStats` method.

**Comment by anwar6953** [comment]
Hmm?

**Comment by anwar6953** [comment]
LGTM

**Comment by gsps1** [comment]
http://builds.cask.co/browse/CDAP-RBT227 - build was green and the last build failed due to known issue in release tracked here https://issues.cask.co/browse/CDAP-2333 - so merging now.

**Pull request merged**